oq-python36/patches/00170-gc-assertions.patch

```diff
@@ -53,23 +53,22 @@ index 0c88603..e3413e8 100644
  }
  #endif
 diff --git a/Lib/test/test_gc.py b/Lib/test/test_gc.py
-index e727499..6efcafb 100644
+index 7e82b24..8ecc3d9 100644
 --- a/Lib/test/test_gc.py
 +++ b/Lib/test/test_gc.py
-@@ -1,10 +1,11 @@
- import unittest
+@@ -2,9 +2,11 @@ import unittest
  from test.support import (verbose, refcount_test, run_unittest,
                            strip_python_stderr, cpython_only, start_threads,
--                          temp_dir, requires_type_collecting)
-+                          temp_dir, import_module, requires_type_collecting)
+                           temp_dir, requires_type_collecting, TESTFN, unlink)
```

Comment:
Curiosity: the previous order is break. Is this precise import order required ?

Reply:
That's an upstream patch taken as it is.